pkg/bugreport/controlplane.go

```diff
@@ -0,0 +1,35 @@
+package bugreport
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/openservicemesh/osm/pkg/constants"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (c *Config) collectControlPlaneLogs() error {
+	pods, err := c.KubeClient.CoreV1().Pods(c.ControlPlaneNamepace).
+		List(context.Background(), v1.ListOptions{
+			LabelSelector: fmt.Sprintf(
+				"%s=%s",
+				constants.OSMAppNameLabelKey,
+				constants.OSMAppNameLabelValue,
+			),
+		})
+	if err != nil {
+		return fmt.Errorf("error getting control plane pods: %w", err)
+	}
+
+	for _, pod := range pods.Items {
+		cmd := []string{"kubectl", "logs", "-n", pod.Namespace, pod.Name}
+		outPath := path.Join(c.rootNamespaceDirPath(), pod.Namespace, rootPodDirName, pod.Name, commandsDirName, strings.Join(cmd, "_"))
+		if err := runCmdAndWriteToFile(cmd, outPath); err != nil {
+			return fmt.Errorf("error writing control pod logs: %w", err)
+		}
```

Comment:
For the application pod reports we output a success line for each generated pod report.
```shell
[+] Collecting information from individual app pods
✓ Collected report for Pod "bookbuyer/bookbuyer-6f655ccfb8-vv485"
✓ Collected report for Pod "bookstore/bookstore-v1-67464975d6-4r87d"
✓ Finished generating report for individual app pods
```
Should we also do the same for the control plane pods?
```
	c.completionSuccess("Collected report for Pod %q", pod)
```

Reply:
Good idea, thanks! Done!